### ISSUE TITLE
Cast system

### ISSUE THREAD

**Issue by rjornetc** [eicdev/QRBattle#28]
Characters can cast any number of chips per turn but only one offensive

**Comment by Criskanda** [comment]
Offensive or defensive, would not be right that someone could use several defensive shield chips in the same turn

**Comment by rjornetc** [comment]
Right: Only one [offensive/defensive] or one offensive and one defensive

**Comment by Criskanda** [comment]
Only one [offensive/defensive], because we must let an opportunity to counterattack.

**Comment by 096acc** [comment]
acta23feb15
- max 1 defensive/offensive chip per turn
- no other limits
